index.js

```diff
@@ -2,8 +2,17 @@ const fs = require('fs');
 
 function AssetFingerprint(initializerDirectory, needsFingerprint = true, fingerprintName) {
   _validateInitializerDirectory();
-
   this.initializerDirectory = initializerDirectory;
+  this.initializerPath = `${this.initializerDirectory}/asset_fingerprint.rb`;
+  
+  if(fingerprintName) {
+    _validateFingerprintName(fingerprintName);
+
+    if(_checkForExistingFingerprint(fingerprintName, this.initializerPath)) {
+      throw new Error('The provided fingerprint name has already been used.');
```

Comment:
This is being thrown for me on secondary runs. I'm testing it by running two independent bundles. The first plugin is setup as a default: `new AssetFingerprintPlugin('config/initializers', true)` which writes `ASSET_FINGERPRINT`. The second build sets the plugin up with the optional argument: `new AssetFingerprintPlugin('config/initializers', true, 'TEST_FINGERPRINT')`. On the first run, it creates the file, adds the default and then the override as expected. But if I run that delegation script again it throws this error. Is there a more deterministic way to test/implement this that you're using?

Reply:
Per our discussion, this was a local testing implementation issue. 👍